test/init-test.js

```diff
@@ -1,8 +1,8 @@
-const chai = require('chai');
-const chaiBN = require('chai-bn');
-const chaiAsPromised = require('chai-as-promised');
+const chai = require("chai");
```

Comment:
Replacing simple quotes by double quotes is weird, is the ESLINT configuration correct?

Reply:
fixed